### PR TITLE
Upgrade to v1.6.1 of the Akamai Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #53
Fixes: #54 
Fixes: #55

### CODE REVIEW

**Comment by stack72** [comment]
@mikhailshilkov the upstream provider had the same *breaking* changes in it in 1.6.0 - https://github.com/akamai/terraform-provider-akamai/blob/master/CHANGELOG.md#160-june-21-2021 so under our breaking change policy, this is something we will accept as a breaking change as well

It's all noted in the CHANGELOG

File: provider/resources.go [hunk 1]
```diff
@@ -97,105 +101,109 @@ func Provider() tfbridge.ProviderInfo {
...
+			"akamai_appsec_ip_geo_protection":               {Tok: makeResource(mainMod, "AppSecIPGeoProtection")},
+			"akamai_appsec_rule":                            {Tok: makeResource(mainMod, "AppSecRule")},
+
+			"akamai_cps_dv_enrollment": {Tok: makeResource(mainMod, "CpsDvEnrollment")},
+			"akamai_cps_dv_validation": {Tok: makeResource(mainMod, "CpsDvValidation")},
+
+			"akamai_networklist_activations":  {Tok: makeResource(mainMod, "NetworkListActivations")},
+			"akamai_networklist_description":  {Tok: makeResource(mainMod, "NetworkListDescription")},
+			"akamai_networklist_network_list": {Tok: makeResource(mainMod, "NetworkList")},
+			"akamai_networklist_subscription": {Tok: makeResource(mainMod, "Subscription")},
```
Comment (mikhailshilkov) [inline provider/resources.go:151]: Would it make sense to call it `NetworkListSubscription` as others above?

**Review by mikhailshilkov — APPROVED** [review]
LGTM with one suggestion

File: provider/resources.go (hunk 1 above)
Comment (stack72) [reply provider/resources.go:151]: Great catch! Thank you - you are correct!

**Pull request merged**